simplecpp.cpp

```diff
@@ -385,7 +385,8 @@ class FileStream : public simplecpp::TokenList::Stream {
         , lastCh(0)
         , lastStatus(0)
     {
-        assert(file != nullptr);
+        if (file != nullptr)
+            throw simplecpp::Output::FILE_NOT_FOUND;
```

Comment:
hmm typically throwing a class is prefered. maybe in the short term this enum will work well.. this is not a pattern we want to start using throughout the code I guess.